x/auth/client/context/utils.go

```diff
@@ -1,25 +0,0 @@
-package context
-
-import (
-	tcmd "github.com/tendermint/tendermint/cmd/tendermint/commands"
-	tmtypes "github.com/tendermint/tendermint/types"
-)
-
-// defaultChainID returns the chain ID from the genesis file if present. An
-// error is returned if the file cannot be read or parsed.
-//
-// TODO: This should be removed and the chainID should always be provided by
-// the end user.
-func defaultChainID() (string, error) {
```

Comment:
Why did we remove this? Did it move somewhere else?

Reply:
I thought that you wanted me to rip this out - were you referring to ripping out my code rather than this method? Can add this back easily if that's the case.

Reply:
Sorry for the confusion. I suggested we make this public and re-use it in your config building code 👍

Reply:
Ah! Will fix :)

Reply:
Encountered a dependency cycle making this public. Will remediate and push.